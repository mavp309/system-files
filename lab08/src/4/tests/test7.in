-3.14 -1.73 4.0 8.2
