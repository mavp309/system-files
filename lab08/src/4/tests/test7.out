0.9600 0.2700
